types/src/ledger_info.rs

```diff
@@ -165,6 +163,29 @@ impl LedgerInfoWithSignatures {
     }
 }
 
+/// Helper function to generate LedgerInfoWithSignature from a set of validator signers used for testing
+pub fn generate_ledger_info_with_sig(
```

Comment:
we should add #[cfg(any(test, feature = "fuzzing"))] to this and any testing related functions

Reply:
Good pointer - fixed.